### PR TITLE
Add and/or update documentation alt text

### LINKED ISSUE #3540: More informative alt text for images
I've noticed that the alt text for all of the images included with markdown in this repository appears to say the same thing: "image". This seems less than helpful.

I'd like if we replaced the alt text with something better, probably along the lines of "image: a more informative of the image content here"

### PR DESCRIPTION
**Pull request by isabela-pf**
# Description

This PR adds or updates existing documentation alt text to be more descriptive. In particular, it describes the images in relation to the surrounding text they illustrate rather than describing them more generally.

We did not get to add or update all alt text in the documentation; this PR contributes only a selection.

If you have any questions about why a change was made, don't hesitate to ask! You may also find help in our discussion on the [collaborative working PR](https://github.com/isabela-pf/napari/pull/2) that helped create this.

## Type of change

This is a change to images in the documentation, but does not require a documentation update as far as I know.

# References

#Fixes #3540

This work was done as a part of an event! You can find more info about the event and what resources we used on the [alt text workshop agenda](https://hackmd.io/vDsWUBqQSxW_7TNxF-RKKA).

# How has this been tested?

- [ ] example: the test suite for my feature covers cases x, y, and z
- [ ] example: all tests pass with my change

## Final checklist:
- [x] My PR is the minimum possible work for the desired functionality
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] If I included new strings, I have used `trans.` to make them localizable.
      For more information see our [translations guide](https://napari.org/docs/dev/guides/translations.html).

### CODE REVIEW

**Review by melissawm — APPROVED** [review]
Test failures are unrelated

File: README.md [hunk 1]
```diff
@@ -2,17 +2,17 @@
...
-[![PyPI](https://img.shields.io/pypi/v/napari.svg)](https://pypi.org/project/napari)
-[![PyPI - Downloads](https://img.shields.io/pypi/dm/napari.svg)](https://pypistats.org/packages/napari)
+[![Code coverage](https://codecov.io/gh/napari/napari/branch/main/graph/badge.svg)](https://codecov.io/gh/napari/napari)
+[![Supported Python versions](https://img.shields.io/pypi/pyversions/napari.svg)](https://python.org)
+[![Python package index](https://img.shields.io/pypi/v/napari.svg)](https://pypi.org/project/napari)
+[![Python package index download statistics](https://img.shields.io/pypi/dm/napari.svg)](https://pypistats.org/packages/napari)
 [![Development Status](https://img.shields.io/pypi/status/napari.svg)](https://en.wikipedia.org/wiki/Software_release_life_cycle#Alpha)
 [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/python/black)
-[![DOI](https://zenodo.org/badge/144513571.svg)](https://zenodo.org/badge/latestdoi/144513571)
+[![D.O.I.](https://zenodo.org/badge/144513571.svg)](https://zenodo.org/badge/latestdoi/144513571)
```
Comment (nclack) [inline README.md]:
```suggestion
[![DOI](https://zenodo.org/badge/144513571.svg)](https://zenodo.org/badge/latestdoi/144513571)
```
I've never seen `D.O.I.` when referencing the doi system. I suspect `DOI` is more correct.

File: docs/guides/layers.md [hunk 2]
```diff
@@ -69,7 +69,7 @@ colors.
 
 For example:
 
-![image: blending](./images/blending.png)
+![napari viewer with an image of a cell. Layer controls are open in left sidebar and blending set to additive.](./images/blending.png)
```
Comment (nclack) [inline docs/guides/layers.md]:
```suggestion
![napari viewer with an image of a cell. Layer controls are open in the left sidebar with the blending set to additive.](./images/blending.png)
```

File: docs/guides/layers.md [hunk 3]
```diff
@@ -105,7 +105,7 @@ distances from the camera according to a maximum intensity projection to create
 the 2D image that is then displayed on the screen. This mode works well for many
 biological images such as these cells growing in culture:
 
-![image: rendering](./images/rendering.png)
+![napari viewer with an image of cells in a culture. Layer controls are open in left sidebar and rendering set to mip.](./images/rendering.png)
```
Comment (nclack) [inline docs/guides/layers.md]:
```suggestion
![napari viewer with an image of cells in a culture. Layer controls are open in the left sidebar and rendering set to mip.](./images/rendering.png)
```

File: docs/tutorials/processing/dask.md [hunk 4]
```diff
@@ -128,7 +128,7 @@ stack = imread("/path/to/experiment/*.tif")
 napari.view_image(stack, contrast_limits=[0,2000], multiscale=False)
 ```
 
-![image: mCherry-H2B showing chromosome separation during mitosis. Collected on a lattice light sheet microscope](../assets/tutorials/dask1.webm)
+![napari viewer with image loaded from dask array showing mCherry-H2B showing chromosome separation during mitosis. Collected on a lattice light sheet microscope.](../assets/tutorials/dask1.webm)
```
Comment (nclack) [inline docs/tutorials/processing/dask.md]:
```suggestion
![napari viewer with image loaded as a dask array showing mCherry-H2B showing chromosome separation during mitosis. Collected on a lattice light sheet microscope.](../assets/tutorials/dask1.webm)
```

File: docs/tutorials/processing/dask.md [hunk 5]
```diff
@@ -248,7 +248,7 @@ but it's surprisingly usable,
 and allows you to preview the result of a relatively complex processing pipeline *on-the-fly*,
 for arbitrary timepoints/channels, while storing *only* the raw data on disk.
 
-![image: same dataset, demonstrating on-the-fly read → deskew → deconvolve → crop](../assets/tutorials/dask2.webm)
+![napari viewer showing same dataset as previous video, demonstrating on-the-fly processing of dask array with sliding through dimensions still preforming reasonably well considering the amount of processing going on as you slice.](../assets/tutorials/dask2.webm)
```
Comment (nclack) [inline docs/tutorials/processing/dask.md]:
```suggestion
![napari viewer showing the same dataset as the previous video, demonstrating on-the-fly processing of a dask array with sliding through dimensions still performing reasonably well considering the amount of processing going on as you slice.](../assets/tutorials/dask2.webm)
```

**Review by nclack — COMMENTED** [review]
I suggested a few small corrections.  One theme was adding "the" in various places. If the suggestions conflict with the consistency of how thing's read, then please ignore them.

**Review by jni — APPROVED** [review]
Thanks @isabela-pf!

I'm going to merge this quickly because I just realised that we still have a bunch of gifs in our images/ folder that were not converted in #4207. Converting those to webm should decrease our lfs usage even more, potentially getting it within our current monthly quota! (Last month our lfs quota only started blocking us a couple of days before the reset.)

**Comment by jni** [comment]
LOL Thanks Nathan for your more careful review! I've applied all your suggestions. I suspect that D.O.I. was to have screen readers read it as the letters rather than the word "doi" in all caps 😂, but also we might remove all those badges as part of #4275, so either way not critical imho.

**Pull request merged**